docs/project/changelog.md

```diff
@@ -172,6 +171,9 @@ async function runPythonAsync(code, globals) {
 
 ### packages
 
+- {{ Update }} Upgrade `scikit-image` to 0.19.1
```

Comment:
```suggestion
- {{ Enhancement }} Upgrade `scikit-image` to 0.19.1
```

It looks like we are using `enhancement` badge for package update

Reply:
Maybe it would make more sense to change them all to `Update`?

Reply:
I agree. And I think we better rename `Update` tag to `Upgrade` or `Version Up`?

Reply:
I agree we should indeed improve the changelog, I think it's simpler to just have a list, rather than an entry per package. Of course, major or long-awaited ones such as scipy still deserve their own changelog. For the rest I would rather spend time on some UI to list packages https://github.com/pyodide/pyodide/issues/977